core/modules/contact/contact.admin.inc

```diff
@@ -22,6 +22,17 @@ function contact_settings_form($form, &$form_state) {
     '#collapsed' => FALSE,
     '#title' => t("Fields"),
   );
+  $form['fields']['phone'] = array(
+    '#type' => 'select',
+    '#title' => t('Phone number'),
+    '#description' => t('Allow users to enter their phone number.'),
+    '#options' => array(
+      'optional' => t('Optional'),
+      'required' => t('Required'),
+    ),
+    '#empty_value' => '',
```

Comment:
Can we please change this one to:

```suggestion
    '#empty_value' => t('Disabled'),
```

...I believe that it makes more sense in this context, as compared to the default `- None -`

Reply:
I disagree. 'None' means there's no field (which is true). 'Disabled' implies that the field is there but cannot be edited (which is not true).

Reply:
Fair enough re "disabled". ...how about "hidden" then? ...was actually going for consistency and was thinking of this:

![image](https://user-images.githubusercontent.com/2423362/98184940-47d6d280-1f14-11eb-98b1-682d5cc5ceba.png)

Reply:
I don't know... 'Hidden' implies the field is added but not visible. I really don't see the problem with 'None'... Maybe bring it up in the issue and see what others think.

Reply:
I believe "none" makes more sense.